.coveragerc

```diff
@@ -3,3 +3,6 @@ relative_files = True
 
 # Use 'source' instead of 'omit' in order to ignore 'tests/unit/__init__.py'
 source = explode
+
+[report]
+fail_under = 70
```

Comment:
Not sure why the coverage service reported failure after a library update.  I've decided to lower the threshold on this lambda because [we basically got it from AWS repo](https://github.com/aws-cloudformation/aws-cloudformation-macros/tree/master/Explode).

Reply:
I'm not sure if this is the issue, it might be because the coverage is decreasing by 0.5%, there are two different settings in the Web UI: `Coverage Threshold for failure` and `Coverage Decrease Threshold for failure`. They were both unset, so I just set them both to be 50% at https://coveralls.io/github/Sage-Bionetworks-IT/cfn-explode-macro/settings (I don't see a coverage.rc option for the decrease threshold setting)

I don't think there's a way to manually re-run coveralls for a PR, but if you update with a new or changed commit then the next run will hopefully pass.